### DISCUSSION TITLE
[Projects beta] Changing tabs order

### DISCUSSION THREAD

**Discussion by atompie** [community/community#7606] [Projects and Issues] [answered]
It would be great to change tabs order.

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/4803

**Comment by azenMatt** [comment, marked as answer]
Hi @atompie  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄
